spec/rails_helper.rb

```diff
@@ -26,6 +26,8 @@
 # If you are not using ActiveRecord, you can remove this line.
 ActiveRecord::Migration.maintain_test_schema!
 
+Dir[Rails.root.join("spec/page_objects/**/*.rb")].each { |f| require f }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.